go/nbs/store.go

```diff
@@ -92,8 +93,21 @@ type LocalStoreFactory struct {
 	maxTables  int
 }
 
+func CheckDir(dir string) error {
+	stat, err := os.Stat(dir)
+	if os.IsNotExist(err) {
+		return fmt.Errorf("Directory does not exist: %s", dir)
```

Comment:
Presumably, `err` is already an error that says `dir` doesn't exist...why not just return that?